deposit/hal/metadata.py

```diff
@@ -27,26 +27,9 @@
 
 from deposit.sword.metadata import addChild
 from deposit.sword.metadata import MetadataFormatter
-from django.utils.translation import ugettext_lazy as _
 from lxml import etree
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/deposit/hal/metadata.py
+++ b/deposit/hal/metadata.py
@@ -142,7 +142,8 @@
 
         abstract = addChild(profileDesc, 'abstract')
         abstract.attrib[XMLLANG_ATTRIB] = 'en'
-        abstract.text = 'No abstract.' if not form.cleaned_data['abstract'] else form.cleaned_data['abstract']
+        abstract.text = 'No abstract.' if not form.cleaned_data[
+            'abstract'] else form.cleaned_data['abstract']
         for record in paper.sorted_oai_records:
             if record.description:
                 abstract.text = record.description
```